state/pgstatestorage.go

```diff
@@ -749,6 +798,19 @@ func (p *PostgresStorage) closeBatch(ctx context.Context, receipt ProcessingRece
 	return err
 }
 
+// UpdateGERInOpenBatch update ger in open batch
+func (p *PostgresStorage) UpdateGERInOpenBatch(ctx context.Context, ger common.Hash, dbTx pgx.Tx) error {
```

Comment:
Please add sanity check to ensure that the batch has no txs in it. And force `dbTx != nil`

Reply:
Also, it would be nice to not only update the GER, but also the timestamp with `time.Now()`